Cargo.toml

```diff
@@ -1,2 +1,13 @@
 [workspace]
 members = ["fission-server"]
+
+[workspace.dependencies]
```

Comment:
just being sure: I tend to only put in workspace-deps for crates in the space that use all of these things. I'm guessing that's how you've planned this, yeah?